docs/bc-registries-audit-notification-workflow.md

```diff
@@ -0,0 +1,107 @@
+# BC Registries Audit Notification Workflow
+
+The following workflow describes the steps to confirm and resolve a BC Registries Audit alert notification. Additional information regarding the associated error condition and its impact, along with details of the steps to resolve can be found below.
```

Comment:
```suggestion
The following workflow describes the steps to confirm and resolve a BC Registries Audit alert notification. Additional information regarding the associated error condition and its impact, along with details of the steps to resolve it can be found below.
```